package main

//Special case of Invalid
//Checcking if return check is done once already seen return in the current scope (it does go checking )


// return invalid 
func main() {
		
	
}

func f () int {
	return 1
	x := 0
	x = 1
}
 